src/mode/modeHandler.ts

```diff
@@ -2065,11 +2065,11 @@ export class ModeHandler implements vscode.Disposable {
   setStatusBarColor(color: string): void {
     vscode.workspace.getConfiguration('workbench').update(
```

Comment:
Consider separating the get and the update into two separate lines for readability.